File: TugasPekan 1 Java/TugasPekanSatu.java
import java.util.Scanner;

public class TugasPekanSatu {
    public static void main(String[] args) {
        int pilih, angka1, angka2, jumlah;

        Scanner input = new Scanner(System.in);

        do {
            System.out.println("Menampilkan tugas kelayar");
            System.out.println("\n 1. Identias\n 2. Kalkulator\n 3. Perbandingan\n 99. Exit\n");
            System.out.print("Silahkan pilih menu dengan ketik angka : ");
    
            pilih = input.nextInt();
    
            if (pilih == 1) {
                System.out.println("Menu Identitas\n");
                System.out.println("Nama : April\nTTL  : xx-xx-xxxx\n");
            } else if (pilih == 2) {
                System.out.println("Menu Kalkulator\n");
                System.out.println("\n 1. Penjumlahan\n 2. Pengurangan\n 3. Perkalian\n 4. Pembagian\n 5. Sisa Bagi\n");
    
                System.out.print("Masukan pilihan dengan ketik angka : ");
                pilih = input.nextInt();
    
                System.out.print("Masukan angka pertama : ");
                angka1 = input.nextInt();
                System.out.print("Masukan angka kedua : ");
                angka2 = input.nextInt();
    
                switch (pilih) {
                    case 1 :
                        jumlah = angka1 + angka2;
                        System.out.println("Hasil Penjumlaha dari : " + angka1 + " + " + angka2 + " = " + jumlah);
                        break;
                    case 2 :
                        jumlah = angka1 - angka2;
                        System.out.println("Hasil Pengurangan dari : " + angka1 + " - " + angka2 + " = " + jumlah);
                        break;
                    case 3 :
                        jumlah = angka1 * angka2;
                        System.out.println("Hasil Perkalian dari : " + angka1 + " x " + angka2 + " = " + jumlah);
                        break;
                    case 4 :
                        jumlah = angka1 / angka2;
                        System.out.println("Hasil Pembagian dari : " + angka1 + " / " + angka2 + " = " + jumlah);
                        break;
                    case 5 :
                        jumlah = angka1 % angka2;
                        System.out.println("Hasil Sisa Bagi dari : " + angka1 + " % " + angka2 + " = " + jumlah);
                        break;
                }
            
            } else if (pilih == 3) {
                System.out.println("Menu Perbandingan");
                System.out.print("Masukan angka pertama : ");
                angka1 = input.nextInt();
                System.out.print("Masukan angka kedua : ");
                angka2 = input.nextInt();
    
                if (angka1 > angka2) {
                    System.out.print(angka1 + " Lebih besar dari " + angka2);
                } else if (angka1 < angka2) {
                    System.out.print(angka1 + " Lebih kecil dari " + angka2);
                } else {
                    System.out.println(" Kedua bilangan sama ");
                }
            }
            
        } while ( pilih != 99 );
    }
}

File: TugasPekan 1 Python/TugasPekanSatu.py


print(" Menampilkan MENU kelayar")
print(" 1. Identitas\n 2. Kalkulator\n 3. Perbandingan\n")

pil = int(input("Silahkan pilih menu : "))

if pil == 1:
    print("Menu Identitas")
    print(" Nama  : Ganteng\n TTL   : xx-xx-xxx\n")

elif pil == 2:
    print("Kalkulator Sederhana Python\n")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")

    pil = int(input("\nMasukan Pilihan Operasi : "))

    x = int(input("Masukan Angka Pertama : "))
    y = int(input("Masukan Angka Kedua : "))

    if pil == 1:
        hasil = x+y
    elif pil == 2:
        hasil = x-y
    elif pil == 3:
        hasil = x*y
    elif pil == 4:
        hasil = x/y
    elif pil == 5:
        hasil = x%y

    print("\nHasil dari : %d" %hasil)

elif pil == 3:
    print("Menu Perbandingan\n")
    x = int(input("Masukan Angka Pertama : "))
    y = int(input("Masukan Angka Kedua : "))

    if (x > y ):
        print(x + " lebih besar dari " + y)
    elif (x < y):
        print(x + "lebih kecil dari " + y)
